src/talks_tweeter.hrl

```diff
@@ -0,0 +1,2 @@
+-record(talk,  {title :: string(), start_time :: calendar:datetime(), end_time :: calendar:datetime(), location :: string()}).
+-record(state, {table :: ets:tab()}).
```

Comment:
State is something internal to the tt_store process. Thus it shouldn't be in the .hrl file. Let's move it back to the tt_store module.